### DISCUSSION TITLE
Can someone provide an example of react-markdown in a Remix application?

### DISCUSSION THREAD

**Discussion by alindsay55661** [remarkjs/remark#992] [Q&A] [answered]
Seems like some people have got it working and others haven't, but no complete code examples I can find. Who can provide a working example of react-markdown in a Remix application?

**Comment by ChristianMurphy** [comment, marked as answer]
Taking a step back, it's worth noting the remix natively include remark and mdx support, and that react-markdown isn't needed in many situations https://remix.run/docs/en/v1/guides/mdx

If you do have a specific use-case for using a more custom react-markdown setup, these recommendations from the remix team may help https://github.com/remix-run/remix/discussions/2594

**Reply by alindsay55661** [reply]
Some how I missed this, thank you!
